### DISCUSSION TITLE
Database

### DISCUSSION THREAD

**Discussion by Miloszzz2** [wailsapp/wails#3267] [Q&A]
Hello, I'm completly new in wails and this kind of desktop apps. I want to ask what is the best approach to connect local postgresql db? Is it possible to use popular orm's like prisma or drizzle?

**Comment by leaanthony** [comment]
Yes and yes! 😉 Anything available for Go is available for Wails.

**Reply by KiddoV** [reply]
@leaanthony,
@Miloszzz2 was talking about 2 frontend techs (Prisma & Drizzle) though, but I would prefer [GORM](https://gorm.io/) 😉, to handle all database stuffs in the backend with Go.
